### ISSUE TITLE
Crashes upon bug upload

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/Instabug-React-Native#349]
Hi can you please resolve that? Noone can send feedback because the app crashes.

### Steps to Reproduce the Problem

```
NSInvalidArgumentException -[__NSCFConstantString allKeys]: unrecognized selector sent to instance 0x1d07dfbf8 
    Frameworks/CoreFoundation.framework/CoreFoundation ___exceptionPreprocess
    /usr/lib/libobjc.A.dylib _objc_exception_throw
    Frameworks/CoreFoundation.framework/CoreFoundation -[NSObject(NSObject) doesNotRecognizeSelector:]
    Frameworks/CoreFoundation.framework/CoreFoundation ____forwarding___
    Frameworks/CoreFoundation.framework/CoreFoundation ___forwarding_prep_0___
    /Users/distiller/project/Instabug/InstabugNetworkLogger/Persistence/Core Data/Models/IBGNetworkLog+CoreDataClass.m:120:24 -[IBGNetworkLog toDictionary]
    /Users/distiller/project/Instabug/InstabugNetworkLogger/Modules/IBGNetworkLoggerPersistenceManager.m:195:29 __82-[IBGNetworkLoggerPersistenceManager loadLogsAsynchronouslyWithCompletionHandler:]_block_invoke
    /Users/distiller/project/Instabug/InstabugCoreDataLayer/IBGDALService.m:9 __43-[IBGDALService readASync:completionBlock:]_block_invoke
    Frameworks/CoreData.framework/CoreData _developerSubmittedBlockToNSManagedObjectContextPerform
    /usr/lib/system/libdispatch.dylib __dispatch_client_callout
    /usr/lib/system/libdispatch.dylib __dispatch_lane_serial_drain$VARIANT$mp
    /usr/lib/system/libdispatch.dylib __dispatch_lane_invoke$VARIANT$mp
    /usr/lib/system/libdispatch.dylib __dispatch_workloop_worker_thread
    /usr/lib/system/libsystem_pthread.dylib __pthread_wqthread
    /usr/lib/system/libsystem_pthread.dylib _start_wqthread
```

Instant crash whenever I tap on the submit (airoplane) icon to send a bug report.

### Expected Behavior
Should not crash.

### SDK Version
"instabug-reactnative": "8.4.2",

### React Native, iOS and Android Versions
"react-native": "0.59.9",
iOS 12.4

### Device Model
iPad Air 2

**Comment by HeshamMegid** [comment]
@SudoPlz we're very sorry you're running into this crash. This issue has been fixed in 8.4.4. Please update to the latest version.

**Comment by SudoPlz** [comment]
Awesome, I'll update, thanks!
